migrations/versions/8c6f4554d24d_create_performance_profile_history.py

```diff
@@ -0,0 +1,33 @@
+"""create performance profile history
+
+Revision ID: 8c6f4554d24d
+Revises: 26a16593ec96
+Create Date: 2022-03-24 13:39:41.695476
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '8c6f4554d24d'
+down_revision = '26a16593ec96'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    try:
+        op.execute(
+            'CREATE TABLE performance_profile_history AS TABLE performance_profile'
```

Comment:
Yes because these fields were present against system record and values for these fields were updated based on latest reported data. That's why you will see that these fields` values are populated ONLY in performance_profile records.

Reply:
Yeah. Also we're not using these fields in our current `/history` endpoint. This would work however.